Formula/gambit-scheme.rb

```diff
@@ -0,0 +1,34 @@
+class GambitScheme < Formula
+  desc "Gambit Scheme is an implementation of the Scheme Language"
+  homepage "http://gambitscheme.org"
+  url "https://github.com/gambit/gambit/archive/v4.9.1.tar.gz"
+  sha256 "667ae2ee657c22621a60b3eda6e242224d41853adb841e6ff9bc779f19921c18"
+
+  depends_on "openssl"
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      --enable-single-host
+      --enable-multiple-versions
+      --enable-default-runtime-options=f8,-8,t8
+      --enable-poll
+      --enable-openssl
+    ]
+
+    inreplace "lib/os_io.c" do |s|
```

Comment:
This bug needs to be reported and fixed upstream.

Reply:
Will remove this.